examples/MatLabAgent_v2/mlaV2/agent.py

```diff
@@ -0,0 +1,178 @@
+"""
+Agent documentation goes here.
+"""
+
+__docformat__ = 'reStructuredText'
+
+import logging
+import sys
+from pprint import pformat
+from volttron.platform.agent import utils
+from volttron.platform.vip.agent import Agent, Core, PubSub, RPC
+
+_log = logging.getLogger(__name__)
+utils.setup_logging()
+__version__ = "0.2"
+
+
+def mlaV2(config_path, **kwargs):
+    """Parses the Agent configuration and returns an instance of
+    the agent created using that configuration.
+
+    :param config_path: Path to a configuration file.
+
+    :type config_path: str
+    :returns: Mlav2
+    :rtype: Mlav2
+    """
+    try:
+        config = utils.load_config(config_path)
+    except StandardError:
+        config = {}
+
+    if not config:
+        _log.info("Using Agent defaults for starting configuration.")
+
+    script_names = config.get('script_names', ["test.py"])
+    script_args = config.get('script_args', [["20"]])
+    topics_to_agents= config.get('topics_to_agents', ["matlab/to_agent/1"])
+    topics_from_agents = config.get('topics_from_agents', "matlab/from_agent/")
+
+
+    return Mlav2(script_names, script_args, topics_to_agents, topics_from_agents, **kwargs)
+
+
+class Mlav2(Agent):
```

Comment:
Please change class name. MatlabV2 or something like that instead of this short form